sqlup-mode.el

```diff
@@ -144,9 +144,8 @@ Other than <RET>, characters are in variable sqlup-trigger-characters."
 (defun sqlup-match-eval-keyword-p (dialect)
   "Return t if the code just before point ends with an eval keyword valid in
 the given DIALECT of SQL."
-  (cl-some 'identity
-           (mapcar #'(lambda (kw) (looking-back (concat kw "[\s\n\r\t]*") 0))
-                   (cdr (assoc dialect sqlup-eval-keywords)))))
+  (cl-some (lambda (kw) (looking-back (concat kw "\\s-*") 0))
```

Comment:
Actually if you aren't particularly attached to common lisp then [`seq.el`](http://endlessparentheses.com/new-on-elpa-and-in-emacs-25-1-seq-el.html) might be the thing to use. It's part of emacs from (I think) version 25 onwards and is available from elpa for older versions. It also supports all kinds of sequences instead of just lists.